keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -171,11 +178,28 @@ def __init__(
                 "a single class is present, the model will always give a score of "
                 "`1` for the single present class."
             )
-        if backbone is None:
-            self.backbone = keras_cv.models.ResNet50V2Backbone()
-        else:
-            self.backbone = backbone
 
+        if backbone is not None and feature_extractor is not None:
+            raise ValueError(
+                "Expected one of `backbone` or `feature_extractor`, but got "
+                f"`backbone={backbone}`"
+                f"`feature_extractor={feature_extractor}`"
+            )
+
+        # Only create a backbone if feature_extractor is None
+        if backbone is None and feature_extractor is None:
+            backbone = keras_cv.models.ResNet50.from_preset('resnet50_imagenet')
```

Comment:
Is an untrained model actually a good backbone? If not I would prefer to change `backbone` to a positional arg and forget the defaults. We can recommend a `backbone` in the docstring.

Reply:
untrained is BAD (0.08~-0.12 MAP).  We can swap it to a positional argument.

Reply:
I've made this required.